smpe/bld/service/current-hold-ACTION.txt

```diff
@@ -0,0 +1,37 @@
+****************************************************************
+* Affected function: Zowe configuration                        *
+****************************************************************
+* Description: Configure external and referrer hostnames       *
+****************************************************************
+* Timing: post-APPLY                                           *
+****************************************************************
+* Part: instance.env                                           *
+****************************************************************
+
+If you are upgrading to zowe [insert Zowe version here]
+from a previous release, 
+and the value of ZOWE_EXPLORER_HOST does not match the host 
+and domain that you put into your browser to access Zowe, 
+then you must update your configuration due to updated 
+referrer-based security. 
+
+You can either:
+
+Define ZWE_EXTERNAL_HOSTS equal to a comma-separated list of 
+hosts you would access Zowe from your browser,
+
+or:
+
+Define ZWE_REFERRER_HOSTS the same way, for the value to be 
+applied specifically for referrer purposes.
+
+New variables are added to instance.env :
+  
+- ZWE_EXTERNAL_HOSTS=host1,host2,...
+# comma-separated list of hosts
```

Comment:
Should this line start with '*' instead of '#'?

Reply:
no, instance.env is read like a shell script, so use shell-style comments

Reply:
Sure, as long as the HOLDDATA parser does not complain.

Reply:
I added this hold data to PR #1538, so I think this PR can be closed. If we want to keep it around we must update it to NOT remove the ptf-bucket, or the PTF build will fail.
Build 5 of #1538 is an SMPE build that will have the hold info.

Reply:
as fold hold data parsing, round brackets ( ) must be paired. and you cannot use /* */ as SMPE sees this as comments (troublesome for adding sample JCL which can hold //*)

Reply:
Noted, thank.  But I don't see any of those deprecated usages in this file, so that's OK, right?

Reply:
Not sure I follow. What do you mean with deprecated usages?

Reply:
I mean I don't see any
`/* ... */`
`( ... )`
Conclusion:  Your file is OK as it now is.